tools/spack-scripting/scripting/cmd/installcosmo.py

```diff
@@ -29,12 +29,22 @@
 
 
 def setup_parser(subparser):
-    arguments.add_common_arguments(subparser, ["jobs"])
+    arguments.add_common_arguments(subparser, ["jobs", "spec"])
 
     subparser.add_argument(
-        "-t", "--test", action="store_true", help="Dev-build with testing"
```

Comment:
this was not used at all, isnt?

Reply:
I don't think so, but it makes sense to me to have this option available in installcosmo.
The ability to test cosmo after installation is crucial at least for all jenkins plan.